malaria24/ona/tests/base.py

```diff
@@ -52,6 +56,27 @@ def mk_mis(self, **kwargs):
     def mk_facility(self, **kwargs):
         return Facility.objects.create(**kwargs)
 
+    def start_randomDate(self):
+        random_year = timezone.now().year
```

Comment:
This is not really all that random. Maybe this variable name can be `current_year`